pkg/ioutil/ioutil.go

```diff
@@ -38,7 +38,7 @@ type WriteOnCloser struct {
 	hasWritten bool
 }
 
-func (w *WriteOnCloser) Write(p []byte) (int, error) {
+func (w *WriteOnCloser) Write(p []byte) (n int, err error) {
```

Comment:
Very minor but named returns not needed...

Reply:
It's just my preferred approach will revert.